app/main.py

```diff
@@ -1,7 +1,40 @@
-def get_weather():
-    # write your code here
-    pass
+import datetime
+import os
+
+import requests
+
+
+def get_weather(requests_city):
+    url = "http://api.weatherapi.com/v1/current.json"
+
+    try:
+        api_key = os.getenv("API_KEY", default=None)
+        parameters = {
+            "key": api_key,
+            "q": requests_city,
+        }
+        response = requests.get(url, params=parameters).json()
+    except TypeError:
+        print("you should give an api_key")
+        return
+
+
+
+    city = response["location"]["name"]
+    country = response["location"]["country"]
+    time = datetime.datetime.strptime(
```

Comment:
We could get the time from the API(by the key ['location']['localtime']), but if it works, than good

Reply:
it seemed to me that it was somewhat illogical, to receive a forecast for a specific time, so I reform it as a date only